src/edu/csus/ecs/pc2/ui/team/CreateTeamsTSV.java

```diff
@@ -0,0 +1,268 @@
+// Copyright (C) 1989-2021 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.ui.team;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import edu.csus.ecs.pc2.core.ParseArguments;
+import edu.csus.ecs.pc2.core.Utilities;
+import edu.csus.ecs.pc2.core.exception.CommandLineErrorException;
+import edu.csus.ecs.pc2.core.imports.clics.TeamAccount;
+
+/**
+ * Read teams.json write teams.tsv
+ * 
+ * @author pc2@ecs.csus.edu
+ * @version $Id$
+ */
+
+public class CreateTeamsTSV {
```

Comment:
This seems like a strange class name to me.  Software Engineering texts typically teach that "classes" represent "things" (they tell students to look for _nouns_ in requirements specifications because those normally indicate classes of objects).  Similarly, they teach that _verbs_ normally represent _actions_ -- methods.   "Create" is a verb, not a Noun.  It seems like this class is a thing which is intended to be used to create Teams TSV files; I would expect it to be named like "TeamsTSVCreator".

Having said that, I don't think it's a _big_ deal; just pointing out that not following conventions can lead to confusion...